.github/ISSUE_TEMPLATE/regression.yaml

```
@@ -49,3 +49,12 @@ body:
       value: kubectl logs -n argo -c wait -l workflows.argoproj.io/workflow=${workflow},workflow.argoproj.io/phase!=Succeeded
     validations:
       required: true
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for submitting this issue! Are you a contributor? If not, have you thought about it? 
+
+        Argo Workflows is seeking more community involvement and ultimately more Reviewers and Approvers to help keep it viable. 
+        We are gauging interest in a potential system in which many companies pledge a little bit of time each to help get more people into these roles. 
+        See [here](https://github.com/argoproj/argo-workflows/issues/12229) for more information. 
+        If you think you or your company might be interested, please add a comment to the issue.
```

```suggestion
        See [#12229](https://github.com/argoproj/argo-workflows/issues/12229) for more information. 
        If you or your company is interested, please add a comment to the issue.
```

- don't use "here" for links, per [k8s style guide](https://kubernetes.io/docs/contribute/style/style-guide/#links)
- more direct wording, per [k8s style guide](https://kubernetes.io/docs/contribute/style/style-guide/#use-simple-and-direct-language)

---

I prefer not to say "is interested" as it implies too much definitive commitment. Is there another way to handle that?

---

Yea I get what you were trying to do with the wording, but IMO "interested" is not a particularly strong commitment to begin with.

"may be interested in getting involved" could be another way of wording it where the word "involved" is similarly low commitment